boards/arm/mimxrt1050_evk/mimxrt1050_evk.dts

```diff
@@ -6,115 +6,63 @@
 
 /dts-v1/;
 
-#include <nxp/nxp_rt.dtsi>
+#include "mimxrt1050_evk_common.dts"
 
 / {
-	model = "NXP MIMXRT1050-EVK board";
-	compatible = "nxp,mimxrt1052";
-
-	aliases {
-		gpio-1= &gpio1;
-		gpio-2= &gpio2;
-		gpio-3= &gpio3;
-		gpio-4= &gpio4;
-		gpio-5= &gpio5;
-		i2c-1 = &i2c1;
-		uart-1 = &uart1;
-		uart-3 = &uart3;
-		led0 = &green_led;
-		sw0 = &user_button;
-		spi-3 = &spi3;
-		eth = &eth;
-	};
-
 	chosen {
-		zephyr,console = &uart1;
-		zephyr,shell-uart = &uart1;
-	};
-
-	sdram0: memory@80000000 {
-		/* Micron MT48LC16M16A2B4-6AIT:G */
-		device_type = "memory";
-		compatible = "mmio-sram";
-		reg = <0x80000000 0x2000000>;
-	};
-
-	leds {
-		compatible = "gpio-leds";
-		green_led: led_0 {
-			gpios = <&gpio1 9 0>;
-			label = "User LD1";
-		};
-	};
-
-	gpio_keys {
-		compatible = "gpio-keys";
-		user_button: button_0 {
-			label = "User SW8";
-			gpios = <&gpio5 0 GPIO_INT_ACTIVE_LOW>;
-		};
-	};
-
-	panel {
-		compatible = "rocktech,rk043fn02h-ct";
-		port {
-			lcd_panel_in: endpoint {
-				remote-endpoint = <&lcd_panel_out>;
-			};
-		};
+		zephyr,flash = &hyperflash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,uart-mcumgr = &uart1;
 	};
 };
 
-arduino_serial: &uart3 {};
+
 
 &flexspi0 {
 	reg = <0x402a8000 0x4000>, <0x60000000 0x4000000>;
 	hyperflash0: hyperflash@0 {
-		compatible = "cypress,s26ks512s";
+		label = "FLEXSPI0";
+		compatible = "soc-nv-flash";
+		/*erase-block-size = <1>;
+		write-block-size = <1>;*/
 		reg = <0>;
 		status = "ok";
 	};
 };
 
-&lcdif1 {
-	status = "ok";
-	port {
-		lcd_panel_out: endpoint {
-			remote-endpoint = <&lcd_panel_in>;
-		};
-	};
-};
-
-&i2c1 {
-	status = "ok";
-
-	fxos8700@1f {
-		compatible = "nxp,fxos8700";
-		reg = <0x1f>;
-		label = "FXOS8700";
-
-		/* Two zero ohm resistors (R263 and R264) isolate sensor
-		 * interrupt gpios from the soc and are unpopulated by default.
-		 * Note that if you populate them, they conflict with LCD and
-		 * Ethernet interrupt gpios.
-		 */
-		int1-gpios = <&gpio1 10 0>;
-		int2-gpios = <&gpio1 11 0>;
-	};
-};
 
-&uart1 {
-	status = "ok";
-	current-speed = <115200>;
-};
+&hyperflash0 {
+    partitions {
+	    compatible = "fixed-partitions";
+	    #address-cells = <1>;
+	    #size-cells = <1>;
+
+	    boot_partition: partition@0 {
+	        label = "mcuboot";
+	        reg = <0x00000000 0x00040000>;
+	        read-only;
+	    };
+
+	    slot0_partition: partition@40000 {
+	        label = "image-0";
+	        reg = <0x00040000 0x01800000>;
+	    };
+
+	    slot1_partition: partition@1840000  {
+	        label = "image-1";
+	        reg = <0x01840000 0x01800000>;
+	    };
+
+	    scratch_partition: partition@3040000 {
+	        label = "image-scratch";
+	        reg = <0x03040000 0x00200000>;
+   		};
+
+		storage_partition: partition@0240000 {
```

Comment:
dedent this node by 1 tab. The address in line also needs to match first address in line 63